wire-ios-data-model/.swiftlint.yml

```diff
@@ -28,6 +28,9 @@ included:
     - Source
     - Tests
 
+excluded:
+    - sourcery
+
```

Comment:
Why is this part of this PR?

Reply:
If I remember correctly to get rid of some SwiftLint Warnings from Data Model. I don't remember why I made the change there and not to develop though.